app/views/layouts/base.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= I18n.locale %>" data-timezone="<%= iana_timezone %>" class="<%= User.current ? "layout-pf layout-pf-fixed" : "no-js"%>">
+<html lang="<%= I18n.locale %>" data-timezone="<%= User.current ? ActiveSupport::TimeZone::MAPPING[User.current.timezone] : nil %>" class="<%= User.current ? "layout-pf layout-pf-fixed" : "no-js"%>">
```

Comment:
Could we keep it as a helper function? Maybe some better name for it, but it should live in the helper.
As well at least deprecate `iana_timezone` as it has no other usage I could find.

Reply:
Found some examples of deprecation in foreman and added that to iana_timezone..Let me know if this looks more aligned to what you were thinking..

Reply:
Yeah, that's what I were thinking :)